individualForm.ttl

```diff
@@ -1,157 +1,206 @@
+# Now hand-edited
+
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
 @prefix ui: <http://www.w3.org/ns/ui#>.
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#>.
 @prefix : <#>.
 
-<http://www.w3.org/2006/vcard/ns#Individual>
-    ui:creationForm <#form1> .
+# Ontology additions or interpretations needed for the form to work well
+
+# The ontolgy file doesnt make them disjoint.  This makes the selctor be a choice.
+vcard:TelephoneType   owl:disjointUnionOf ( vcard:Cell vcard:Home vcard:Work) .
+vcard:Type   owl:disjointUnionOf (vcard:Home vcard:Work) . # for email
+
+# Better field labels
+vcard:Cell ui:label "mobile"@en . # app will imake nitial caps if nec
+vcard:hasAddress ui:label "address"@en .
+vcard:bday ui:label "born"@en.
+vcard:hasEmail ui:label "email"@en .
+vcard:hasTelephone ui:label "phone"@en .
+vcard:note ui:label "notes"@en .
 
+#  The forms themselves
 
-<#form1>
+vcard:Individual
+    ui:creationForm :form1 .
+
+# The addressComment etc fields with a ccomment before each type of field
+# were originally partyly because the labels on the fields were clumsy like "hasAddress".
+# This is fixed by adding the ui:label to the properties above, so lets try
```

Comment:
```suggestion
# The addressComment, etc., fields with a comment before each type of field
# were originally partly because the labels on the fields were clumsy like "hasAddress".
# This is fixed by adding the ui:label to the properties above, so let's try
```